contrib/openssl-cmake/CMakeLists.txt

```diff
@@ -1145,9 +1139,10 @@ set(CRYPTO_SRC
     ${OPENSSL_SOURCE_DIR}/providers/implementations/signature/sm2_sig.c
     ${OPENSSL_SOURCE_DIR}/providers/implementations/storemgmt/file_store.c
     ${OPENSSL_SOURCE_DIR}/providers/implementations/storemgmt/file_store_any2obj.c
-    ${OPENSSL_SOURCE_DIR}/providers/legacyprov.c
     ${OPENSSL_SOURCE_DIR}/providers/nullprov.c
     ${OPENSSL_SOURCE_DIR}/providers/prov_running.c
+    ${OPENSSL_SOURCE_DIR}/ssl/record/methods/tls_pad.c
```

Comment:
tls_pad.c and ssl3_cbc.c are part of `SSL_SRC`, l. 1374/1375 already.
Could you please remove both files from there? (I verified that the code will still build for static/dynamic link).

Reply:
Done.